example/.gitignore

```diff
@@ -1,71 +1,9 @@
-# Miscellaneous
-*.class
-*.lock
-*.log
-*.pyc
-*.swp
-.DS_Store
-.atom/
-.buildlog/
-.history
-.svn/
-
-# IntelliJ related
-*.iml
-*.ipr
-*.iws
-.idea/
-
-# Visual Studio Code related
-.vscode/
-
-# Flutter/Dart/Pub related
-**/doc/api/
+# Files and directories created by pub
```

Comment:
I'd just update to the latest .gitignore template from flutter.   A lot of the defaults are useful (like the .vscode, etc)